test/unit/utils/logger.test.ts

```
@@ -56,13 +56,9 @@ function findLogByEvent(logs: JsonLog[], event: string): JsonLog | undefined {
 }
 
 describe('Logger Config', () => {
-    afterEach(() => {
-        vi.unstubAllEnvs();
-    });
-
     describe('environment config', () => {
         it('should disable logging in test environment', () => {
-            vi.stubEnv('NODE_ENV', 'testing');
+        vi.stubEnv('NODE_ENV', 'testing');
```

ESLint

[error] Expected indentation of 12 spaces but found 8.

(indent)